DashKitUI/Views/NotificationSettingsView.swift

```diff
@@ -13,7 +13,7 @@ import HealthKit
 
 struct NotificationSettingsView: View {
     
-    static let expirationReminderHoursAllowed = 1..<30
+    static let expirationReminderHoursAllowed = 1..<25
```

Comment:
[nit] Consider `1...24` as more readable 1 to 24 hours. The `1..<25` could be read that 24.999 hours is valid.